tests/integration/relations/test_backend_database.py

```diff
@@ -57,34 +59,39 @@ async def test_create_backend_db_admin_legacy_relation(ops_test: OpsTest):
         wait_for_relation_joined_between(ops_test, PG, PGB)
         await ops_test.model.wait_for_idle(apps=[PGB, PG], status="active", timeout=1000),
 
-        userlist = await get_userlist(ops_test)
         cfg = await get_cfg(ops_test)
-        pgb_user = f"relation_id_{relation.id}"
-        pgb_password = userlist[pgb_user]
+        logging.info(cfg.render())
+        pgb_user, pgb_password = await get_backend_user_pass(ops_test, relation)
         assert pgb_user in cfg["pgbouncer"]["admin_users"]
+        assert cfg["pgbouncer"]["auth_query"]
 
         await check_database_users_existence(ops_test, [pgb_user], [], pgb_user, pgb_password)
 
         # Remove relation but keep pg application because we're going to need it for future tests.
         await ops_test.model.applications[PG].remove_relation(
             f"{PGB}:{RELATION}", f"{PG}:database"
         )
+        pgb_unit = ops_test.model.applications[PGB].units[0]
+        logging.info(await get_app_relation_databag(ops_test, pgb_unit.name, relation.id))
```

Comment:
is this meant to test something? If not testing maybe remove it

Reply:
Nope, more logging so the state of the application at this point is easily observable.